.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: ['1.6', '1']  # Test against LTS
+        version: ['1.10', '1']  # Test against LTS
```

Comment:
The new SDPA version does not work on v1.6 but v1.6 is still marked as compatible in `Project.toml`. So now the new SDPA version installs in v1.6 and breaks.

Reply:
I thought I bumped it but I went to fast, let's update it on the registry